gems/pending/fs/ext4/superblock.rb

```diff
@@ -223,6 +223,14 @@ def inodeSize
       isDynamic? ? @sb['inode_size'] : INODE_SIZE
     end
 
+    def isEnabled64Bit?
+      gotBit?(@sb['incompat_flags'], ICF_64BIT)
+    end
+
+    def groupDescriptorSize
+      isEnabled64Bit? ? @sb['group_desc_size'] : GDE_SIZE 
```

Comment:
Is `@sb['group_desc_size']` ever not set? For example, if `isEnabled64Bit?` is `false` is `@sb['group_desc_size']` set to 32, or is it undefined. If it's always set to the proper value, there's no need for the conditional and we can always just return the value of `@sb['group_desc_size']`.